src/main/java/frc/robot/subsystems/ShooterTest.java

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import com.ctre.phoenix6.controls.VelocityVoltage;
+import com.ctre.phoenix6.hardware.TalonFX;
+
+public class ShooterTest extends SubsystemBase {
```

Comment:
Good use of playing around with code! I do suggest making a `prototyping` package under `subsystems` and `commands`. This way you don't accidentally put a prototyping subsystem or command into production code, since it will stand out better in a PR that this is only for prototyping.